src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+import utilPackages.WaitClass;
+
+import org.openqa.selenium.support.ui.Select;
+
+public class CreateOrderPage extends BasePage {
+	
+	public CreateOrderPage(WebDriver webdriver) {
+		super(webdriver);
+	}
+
+	Logger log = Logger.getLogger(CreateOrderPage.class);
+
+	PropertyValExtractors p = new PropertyValExtractors();
+	Actions actions = new Actions(driver);
+	String sheetName = "CreateOrder";
+	String xlsxName = "/ProductDependency_TestData.xlsx";
+
+	@FindBy(how = How.XPATH, using = "//input[@name='j_username']")
+	private WebElement enterLoginID;
+
+	/**
+	 * Method to enter login ID.
+	 * 
+	 * @throws Exception
+	 */
+	public void enterLoginID() throws Exception {
+		CreateOrderPage sp = new CreateOrderPage(driver);
+		log.info("Verifying the Login ID is available or not");
+		Assert.assertTrue(enterLoginID.isDisplayed());
+		enterLoginID.sendKeys(BasePage.getCellData(xlsxName, sheetName, 0, 0));
+	}
+
+	@FindBy(how = How.XPATH, using = "//input[@name='j_password']")
+	private WebElement enterPassword;
+
+	/**
+	 * Method to enter Password.
+	 * 
+	 * @throws Exception
+	 */
+	public void enterPassword() throws Exception {
+		CreateOrderPage sp = new CreateOrderPage(driver);
+		log.info("Verifying the First Name is available or not");
+		Assert.assertTrue(enterPassword.isDisplayed());
+		enterPassword.sendKeys(BasePage.getCellData(xlsxName, sheetName, 1, 0));
+
+	}
+
+	private WebElement selectCompany;
+
+	/**
+	 * Method to select Company.
+	 * 
+	 * @throws IOException
+	 */
+	public void selectCompany() throws Exception {
+		CreateOrderPage sp = new CreateOrderPage(driver);
+		WebElement Companyelement = driver.findElement(By.xpath("//select[@name='j_client_id']"));
+		Select se = new Select(Companyelement);
+		se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 2, 0));
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//a[@class='submit save']")
+	private WebElement clickLoginButton;
+	/**
+	 * Method to Click on Save Changes Button
+	 */
+	public void clickLoginButton() {
+		CreateOrderPage sp = new CreateOrderPage(driver);
+		log.info("Verifying the login button is available or not");
+		Assert.assertTrue(clickLoginButton.isDisplayed());
+		clickLoginButton.click();
+	}
+
+	@FindBy(how = How.XPATH, using = "//*[@id='menu.link.customers']/a")
+	private WebElement clickCustomerTab;
+
+	/**
+	 * Method to click on Customer tab after successful login.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickCustomerTab() throws IOException {
+		log.info("Click on Products Tab after successful login");
+		JavaScriptExec.sleep();
+		Assert.assertTrue(clickCustomerTab.isDisplayed());
+		clickCustomerTab.click();
+	}
+	
+		// @FindBy(how=How.XPATH, using="//a[@class='cell double']//*[text()='Customer
+		// A']")
+		private WebElement clickCustomerName;
+
+		/**
+		 * Method to click on customer name.
+		 * 
+		 * @throws IOException
+		 */
+		public void clickCustomerName() throws IOException {
```

Comment:
@mayankchhajer Please define the method correctly. Define the Xpath and then use that WebElement in the method and if Xpath's is not used then please remove the xpath as well as it WebElement.